src/main/java/com/example/javaitmo2/service/CarService.java

```diff
@@ -23,7 +29,24 @@ public CarService(CarRepository repository, ModelMapper modelMapper) {
     }
 
     public ArrayList<CarResponse> getList() {
```

Comment:
Its better to use Interface as return type as much as possible, List or Collection in this case

Reply:
++